src/main/java/programminglife/controller/BookmarkController.java

```diff
@@ -287,7 +287,7 @@ private static void checkFile(String fileName) {
 
                 transformer.transform(source, result);
 
-            } catch (ParserConfigurationException | TransformerException e) {}
+            } catch (ParserConfigurationException | TransformerException e) { }
```

Comment:
I included this change in the PR for general fixes, so this will generate a merge conflict.